Added travis fix for GEOS-7202

BerryDaniel opened the pull request:
Note: In order to locally have the build pass all tests you will need to add the gt-csv-13.jar file to your lib directory.

simboss commented:
Ciao Daniel,
are we sure we need this workaround? [GEOS-7207](https://osgeo-org.atlassian.net/browse/GEOS-7207) has been fixed yesterday and the importer extension now incude the relevant jar from GeoTools.

simboss commented:
Aside, 
this is the kind of thing that should be communicated to the GS team rather than doing a local workaround otherwise the fix will be never go mainstream and we might pollute gsimporter with workarounds :)

BerryDaniel commented:
The work around was only added as a place holder until the GS team resolved GEOS-7207. I didn't feel reporting this as a new issue since it was already reported and being worked on. In regards to the extension.. as of last night the jar file was still missing from ares, but it is now present in the snapshot extension.. I have removed the comments in the readme and modified the travis configuration. If in some way you feel I mishandled this, please let me know and if I did.. I apologize in advance :)

simboss commented:
No worries, I am just trying to keep communication up :)
